CRM/Contribute/Page/ContributionRecur.php

```diff
@@ -75,6 +75,34 @@ public function view() {
 
     $this->assign('recur', $contributionRecur);
 
+    $templateContribution = CRM_Contribute_BAO_ContributionRecur::getTemplateContribution($this->getEntityId());
+
+    $lineItems = [];
+    $displayLineItems = FALSE;
+    if (!empty($templateContribution['id'])) {
+      $lineItems = [CRM_Price_BAO_LineItem::getLineItemsByContributionID(($templateContribution['id']))];
+      $firstLineItem = reset($lineItems[0]);
+      if (empty($firstLineItem['price_set_id'])) {
+        // CRM-20297 All we care is that it's not QuickConfig, so no price set
+        // is no problem.
+        $displayLineItems = TRUE;
+      }
+      else {
+        try {
+          $priceSet = civicrm_api3('PriceSet', 'getsingle', [
+            'id' => $firstLineItem['price_set_id'],
+            'return' => 'is_quick_config, id',
+          ]);
+          $displayLineItems = !$priceSet['is_quick_config'];
```

Comment:
@jaapjansma Is there any reason not to show line items when `is_quick_config = TRUE`? It is true that there will only be one lineitem but I think that is ok?
Eg. this is what I see with a quick config if I ignore is_quick_config:
![image](https://user-images.githubusercontent.com/2052161/120302847-2e3d1400-c2c6-11eb-804b-71204dde46e7.png)

Reply:
I have copied this bit from `CRM_Contribute_Form_ContributionView` 
I did not fully understand the whole code I have copied. So my assumption is that if I change it here, it should also be changed in `CRM_Conribute_Form_ContributionView` as we are basically using the same data and showing in the same way but on different screens. What do you think? 

See: https://github.com/civicrm/civicrm-core/blob/master/CRM/Contribute/Form/ContributionView.php#L129

Reply:
I have solved this only for contribution recur. It will probably need a different PR for the contribution view.